<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>751</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QTableWidget" name="table_2"/>
     </item>
     <item row="1" column="0">
      <widget class="QTableWidget" name="table_4"/>
     </item>
     <item row="1" column="1">
      <widget class="QTableWidget" name="table_5"/>
     </item>
     <item row="0" column="2">
      <widget class="QTableWidget" name="table_3"/>
     </item>
     <item row="0" column="0">
      <widget class="QTableWidget" name="table_1"/>
     </item>
     <item row="2" column="0">
      <widget class="QTreeView" name="tree_7"/>
     </item>
     <item row="2" column="1">
      <widget class="QTreeView" name="tree_8"/>
     </item>
     <item row="1" column="2">
      <widget class="QTreeView" name="tree_6"/>
     </item>
     <item row="2" column="2">
      <widget class="QTreeView" name="tree_9"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
